deploy-manage/tools/snapshot-and-restore.md

```diff
@@ -143,20 +143,22 @@ You can’t restore a snapshot to an earlier version of {{es}}. For example, you
 
 Any index you restore from a snapshot must also be compatible with the current cluster’s version. If you try to restore an index created in an incompatible version, the restore attempt will fail.
 
-| Index creation version | 6.8 | 7.0–7.1 | 7.2–7.17 | 8.0–8.2 | 8.3–8.17 |
-|------------------------|-----|---------|---------|---------|---------|
-| 5.0–5.6               | ✅   | ❌       | ❌       | ❌       | ✅ [^1^](#footnote-1)   |
-| 6.0–6.7               | ✅   | ✅       | ✅       | ❌       | ✅ [^1^](#footnote-1)   |
-| 6.8                   | ✅   | ❌       | ✅       | ❌       | ✅ [^1^](#footnote-1)   |
-| 7.0–7.1               | ❌   | ✅       | ✅       | ✅       | ✅       |
-| 7.2–7.17              | ❌   | ❌       | ✅       | ✅       | ✅       |
-| 8.0–8.17              | ❌   | ❌       | ❌       | ✅       | ✅       |
+| Index creation version | 6.8 | 7.0–7.1 | 7.2–7.17 | 8.0–8.2 | 8.3–8.18 | 9.0 |
+|------------------------|-----|---------|---------|---------|---------|-----|
+| 5.0–5.6               | ✅   | ❌       | ❌       | ❌       | ✅ [^1^](#footnote-1)   | ✅ [^1^](#footnote-1) |
+| 6.0–6.7               | ✅   | ✅       | ✅       | ❌       | ✅ [^1^](#footnote-1)   | ✅ [^1^](#footnote-1) |
+| 6.8                   | ✅   | ❌       | ✅       | ❌       | ✅ [^1^](#footnote-1)   | ✅ [^1^](#footnote-1) |
+| 7.0–7.1               | ❌   | ✅       | ✅       | ✅       | ✅       | ✅ [^1^](#footnote-1) [^2^](#footnote-2) |
+| 7.2–7.17              | ❌   | ❌       | ✅       | ✅       | ✅       | ✅ [^1^](#footnote-1) [^2^](#footnote-2) |
```

Comment:
imo the hyperlinks for the footnotes are overkill because they're right below the table (clicking feels weird because it doesn't bring you somewhere "new")

if you pull out the hyperlinks, you might want to comma-separate the superscript so it's clear there are two footnotes

Reply:
I actually have no idea how to remove the hyperlinks for the footnotes -- I didn't think we can control that, just seems like it's baked in to how the footnotes work in markdown. Do you know how to do that?